aws-rds-dbinstance/src/test/java/software/amazon/rds/dbinstance/TranslatorTest.java

```diff
@@ -316,8 +316,9 @@ public void test_modifyReadReplicaRequest_parameterGroupNotSet() {
     }
 
     @Test
-    public void test_restoreFromSnapshotRequest_storageType_shouldBeSetOnUpdate() {
+    public void test_restoreFromSqlServerSnapshotRequest_storageType_shouldBeSetOnUpdate() {
```

Comment:
Nice addition on the coverage 👍 
This should help us along and reduce our operational cost over time.